iib/web/api_v1.py

```diff
@@ -481,10 +485,43 @@ def _get_user_queue(serial: Optional[bool] = False) -> Optional[str]:
     else:
         labeled_username = f'PARALLEL:{username}'
 
-    queue = flask.current_app.config['IIB_USER_TO_QUEUE'].get(labeled_username)
-    if not queue:
-        queue = flask.current_app.config['IIB_USER_TO_QUEUE'].get(username)
-    return queue
+    index_queue = flask.current_app.config['IIB_USER_TO_QUEUE'].get(labeled_username)
+    if not index_queue:
+        index_queue = flask.current_app.config['IIB_USER_TO_QUEUE'].get(username)
+
+    if index_queue is None:
+        return index_queue
+    elif isinstance(index_queue, str):
+        # keep original behavior for IIB_USER_TO_QUEUE of type dict[str,str])
+        return index_queue
```

Comment:
Minor: maybe we can club this if and elif

Reply:
The structure changed, please review